docs/servers/add-existing-server.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Add an existing Server
+intro: If you have servers provisioned outside of your desired environment on Devopness, you can still connect them to your Devopness account and take the most of the Devopness infrastructure management features.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+# Add an existing Server
+1. On Devopness, navigate to a project then select an environment
+2. If there's no other server in the selected environment, just click `Add Server`
+3. If a list of resource cards is displayed, find the `Servers` card
+4. Click `View` in the `Servers` card to see a list of existing `Servers`
+5. On the upper-right corner of the list click `ADD SERVER`
+6. On the `Cloud Provider` drop-down menu choose `VPS/Self Hosted`
+7. Provide the `Public IP address` of your existing server
+6. Enter the `SSH Port` to be used to connect to the server
```

Comment:
@plantaz it seems more logical to warn user about the risk of adding an existing server after the `IP address` has been informed. Do you agree?
```suggestion
6. Enter the `SSH Port` to be used to connect to the server
> Note: we do not recommend connecting servers that are already being used in production. Server's existing configuration will be lost and replaced by Devopness during the server setup process
```